Tensile/BenchmarkProblems.py

```diff
@@ -43,6 +43,41 @@
 from .SolutionWriter import SolutionWriter
 from .TensileCreateLibrary import writeSolutionsAndKernels, writeCMake, buildObjectFileNames
 
+
+def generateForkedSolutions (problemType, forkedParameters, commonSolutionParameters):
```

Comment:
I'd like to see this function replace the loop below (line 203/238 of this file) instead of duplicating that functionality.

Reply:
This is actually something I somehow missed in my refactoring. I look into using the Python generator functions.

I want to minimize the amount of refactoring here, we can do more extensive refactoring in future iterations, as time permits.